src/integration/resources/inprocess/coordinator_test.go

```diff
@@ -73,15 +77,49 @@ func TestNewEmbeddedCoordinatorNotStarted(t *testing.T) {
 	require.Error(t, err)
 }
 
-func TestM3msgTopicFunctions(t *testing.T) {
-	dbnode, err := NewDBNodeFromYAML(defaultDBNodeConfig, DBNodeOptions{})
+func TestCoordinatorAPIs(t *testing.T) {
```

Comment:
Consolidated some tests under a single test to reduce cost of spinning up a dbnode and coord each time.